lang/cs/Org.Apache.REEF.Network.Tests/NetworkService/NetworkServiceTests.cs

```diff
@@ -156,10 +156,11 @@ public void TestNetworkServiceTwoWayCommunication()
                  nameServiceAddr)
              .BindImplementation(GenericType<INameClient>.Class, GenericType<NameClient>.Class)
              .Build();
-
-            var nameClient = TangFactory.GetTang().NewInjector(nameserverConf).GetInstance<NameClient>();
+            var injector = TangFactory.GetTang().NewInjector(nameserverConf);
+            var nameClient = injector.GetInstance<NameClient>();
+            var remoteManagerFactory = injector.GetInstance<IRemoteManagerFactory>();
             return new NetworkService<string>(networkServicePort,
```

Comment:
Agreed. We'd need the .NET side for [REEF-290](https://issues.apache.org/jira/browse/REEF-290). This PR is just a small step in that direction.